src/components/controls/controls.vue

```diff
@@ -118,70 +118,4 @@ export default {
 };
 </script>
 
-<style lang="scss">
-$border-color: rgba(0, 0, 0, 0.125);
-$controls-column-max-width: 265px;
-$controls-column-compressed-max-width: 95px;
-$controls-transition: 0.3s;
-
-.modeler {
-  .controls {
-    z-index: 1;
-
-    &-column {
-      max-width: $controls-column-max-width;
-      transition: all $controls-transition ease-out;
-
-      .list-group {
-        &:last-child {
-          border-bottom: 1px solid $border-color;
-        }
-      }
-    }
-
-    &-column-compressed {
-      max-width: $controls-column-compressed-max-width;
-      transition: all $controls-transition ease-in;
-    }
-  }
-}
-
-.card-header {
-  background: #f7f7f7;
-}
-
-.list-group-item:first-child {
-  border-top: 0;
-}
-
-.tool {
-  cursor: grab;
-  user-select: none;
-  font-size: 0.875rem;
-}
-
-.is-dragging {
-  background: #3397e1;
-  color: white;
-  position: absolute;
-  z-index: 10;
-  box-shadow: 5px 5px 8px 0 #0000004a;
-  cursor: grabbing;
-  padding: 0.5rem;
-
-  &.no-drop {
-    opacity: 0.8;
-    cursor: no-drop;
-  }
-
-  &:hover {
-    background-color: #3397e1;
-    color: white;
-  }
-}
-
-.tool-icon {
-  width: 1.5rem;
-  pointer-events: none;
-}
-</style>
+<style lang="scss" src="./controls.scss" scoped />
```

Comment:
The previous styles weren't scoped so I would make sure this doesn't remove styling from other areas.